Phonebook.Frontend/src/app/shared/components/user/user-list-card/user-list-card.component.html

```diff
@@ -0,0 +1,71 @@
+<mat-card [routerLink]="['/user', person.Id]" class="mat-ripple list-card">
+    <mat-card-header *ngIf="actionButtonIcon">
+        <div class="pb-right-top-action">
+            <button propagationStop mat-icon.button [ngClass]="actionButtonClasses" (click)="buttonClicked()" class="close">
+                <mat-icon>{{ actionButtonIcon }}</mat-icon>
+            </button>
+        </div>
+    </mat-card-header>
+    <mat-card-content>
```

Comment:
@DanielHabenicht maybe it is also cool if we could use the settings from the search for all cards?  What do you think about this. Should I create a new feature about this?

Reply:
Already opened #287

Reply:
@mschwrdtnr  no not like this. I mean that all information/columns that are displayed in the search also displayed in the dashboard. So I as user only must change one setting and all my views looks the same.

Reply:
okay, sounds cool, but i think that the most don't want to have the same details on favorites and normal search 🤔 

I can imagine just a small checkbox in settings or in "Configure Table Columns" smth like "Same information like Search-Columns".. but i am not sure about the usefulness.

Reply:
I think this should be discussed in another issue, please open one @paule96 
I think making them configurable would help, but the cards are generally displaying less information than in the table so they should be configurable separately